opensplice_cmake_module/env_hook/opensplice.sh.in

```diff
@@ -1,9 +1,15 @@
-export OSPL_HOME=@OpenSplice_HOME@
+fi [ -z "$OSPL_HOME" ]; then
+  export OSPL_HOME=@OpenSplice_HOME@
```

Comment:
Since this package has been built against OpenSplice in `@OpenSplice_HOME@` it might be problematic to later run against a different OpenSplice installation. Therefore I would argue this environment file should always set the `OSPL_HOME` to point to the OpenSplice version it was built against.

Same for `OSPL_TMPL_PATH` below.